project.conf

```diff
@@ -0,0 +1,11 @@
+
+# This is the configuration used locally.
+# This should not be commited (by default it is ignored by git)
+
+[withings_api_example]
+account_withings_url = https://account.withings.com
+wbsapi_withings_url = https://wbsapi.withings.net
+client_id = 0f985c9465c5295a2b5f3d05ee48c0295f52dbff3d6004b83b317c268cff8eab
```

Comment:
@GradientCrux Looks like you have upload your secret keys. I do not think this is a good idea